editor/plugins/node_3d_editor_plugin.h

```diff
@@ -388,6 +399,28 @@ class Node3DEditorViewport : public Control {
 	void reset_fov();
 	void scale_cursor_distance(real_t scale);
 
+	struct ShortcutCheckSet {
+		bool mod_pressed = false;
+		bool shortcut_not_empty = true;
+		int input_count = 0;
+		ViewportNavMouseButton mouse_preference = NAVIGATION_LEFT_MOUSE;
+		NavigationMode result_nav_mode = NAVIGATION_NONE;
+
+		ShortcutCheckSet(){};
```

Comment:
```suggestion
		ShortcutCheckSet() {}
```

Reply:
Unfortunately clang format removes the space when I try to add it... and I get a warning when I leave out the default constructor. Any ideas?

Reply:
It doesn't if you follow it properly, you didn't remove the `;`